packages/xpetra/test/Matrix/Matrix_UnitTests.cpp

```diff
@@ -241,7 +241,6 @@ namespace {
     Teuchos::RCP<const Teuchos::Comm<int> > comm = getDefaultComm();
     const size_t numLocal = 10;
     const size_t INVALID = Teuchos::OrdinalTraits<size_t>::invalid(); // TODO: global_size_t instead of size_t
```

Comment:
Just a commentary on `global_size_t`: It was a nice thought (32-bit machines would need something bigger than `size_t` to address All The Things), but we already had types for local indices and global indices.  `size_t` and `global_size_t` just create build warnings and confusion.